### PR TITLE
Rename fillIn to enter

### PR DESCRIPTION
**Pull request by alexdavid**
@kevgo @dhakiki please review

Implements #18

### CODE REVIEW

**Comment by dhakiki** [comment]
:+1:

File: src/commands/enter.coffee [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Nifty = require '../..'
+nodeify = require 'nodeify'
+
+
+module.exports = new Nifty.Command 'enter', (text, {into}, done) ->
+  promise = @findElement(into).sendKeys text
```
Comment (dhakiki) [inline src/commands/enter.coffee]: Super straightforward! I like this new syntax a lot. I keep thinking of the edge case where someone uses it wrong and it explodes. Maybe some more error checking can be useful on these commands.

Comment (alexdavid) [inline src/commands/enter.coffee]: Yeah, I think good error messages on improper usage of commands would be incredibly valuable.

I'll open a new PR with some feature tests testing bad data and expecting sane, readable errors

File: features/enter.feature [hunk 2]
```diff
@@ -8,7 +8,7 @@ Feature: Sending keypresses to a webpage
     Given I am on a page with the HTML "<textarea></textarea>"
     When I run
       """
-      niftyBrowser.fillIn 'textarea', with: 'hello browser'
+      niftyBrowser.enter 'hello browser', into: 'textarea'
```
Comment (kevgo) [inline features/enter.feature]: This reads sooooo much better now!

File: src/commands/enter.coffee (hunk 1 above)
Comment (kevgo) [inline src/commands/enter.coffee]: +1

**Comment by kevgo** [comment]
:ship: huge improvement!

**Pull request closed without merging**